tcms/testplans/static/testplans/js/get.js

```diff
@@ -103,3 +117,100 @@ function getTestCaseExpandArea(row, testCase) {
         }
     });
 }
+
+function attachEvents(testPlanId, permissions) {
+    if (permissions['perm-change-testcase']) {
+        // update default tester
+        $('.js-test-case-menu-tester').click(function(ev) {
+            // todo this prompt need to be translated
+            var email_or_username = window.prompt('Please type new email or username');
+            if (!email_or_username) {
+                return;
+            }
+            const testCaseId = getCaseIdFromEvent(ev);
+
+            // default_tester can be update only if we pass an user id
+            jsonRPC('TestCase.update', [testCaseId, {'default_tester': email_or_username}], function(res) {
+
+            });
```

Comment:
Note: none of these refreshes the actual row that it updates! Maybe add as TODO, idk.